pennylane/tape/qscript.py

```diff
@@ -365,6 +373,7 @@ def _update(self):
         # The following line requires _par_info to be up to date
         self._update_trainable_params()  # Updates the _trainable_params; O(1)
 
+        self._update_measured_wires()
```

Comment:
this method is admittedly misleading, but by default, `_update` is called on tape construction, so putting this here would make this evaluation eager (non-lazy).
```suggestion
```

Reply:
I've accepted your suggested change.